Reflections/Week_1_Reflection.md

```diff
@@ -0,0 +1,9 @@
+**Will Pratt**
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+* I have managed to translate my proof of concept to classes I created myself for more of an object oriented programming approach. I have also begun implementing the rooms class from adventurelib within my project.

```

Comment:
Awesome! I've mentioned this before but I'd like to reiterate; putting a graph together on paper (or even in draw.io) would likely be highly beneficial for figuring out how rooms "connect" to each other, as well as room states!